pkgs/development/ruby-modules/tool/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv }@defs:
```

Comment:
Is this file supposed to be in the top-level? I don't see any references to it yet.